spec/models/date_range_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'spec_helper'
+
+RSpec.describe DateRange do
+
+  describe '#dates' do
+      let(:conference) { FactoryGirl.build_stubbed(:conference)}
+    
+    it 'creates a date range for calling Models' do
+      expect(conference.dates).to be_an_instance_of DateRange
+    end
+  end
+  
+  describe '#compact' do
+    let (:conference)  { FactoryGirl.build_stubbed(:conference) }
+    subject { described_class.new(start_date: Date.parse('2015-12-31'), end_date: Date.parse('2015-12-31')) }
+
+    it 'responds to other models' do
```

Comment:
This test appears slightly *"off topic"*, as it does not really test the `DateRange` class. In my opinion all it does, is to see if `#compact` is a public method on a `DateRange`, which is already obvious through all the other specs in this block... So I'd vote for skipping this all remove the conference here completely.